CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ else ()
 endif ()
 
 # Enable parallel GCC algorithms (needed until there is relable support for C++17 from compilers)
```

Comment:
I don't mind leaving this commented out instead of removing it, but I would edit the comment to reflect the current state and explain why this is disabled.

Reply:
I think it can be axed. fixing...